cassandra/src/main/scala/filodb.cassandra/columnstore/CassandraColumnStore.scala

```diff
@@ -166,6 +168,13 @@ extends CachedMergingColumnStore with CassandraColumnStoreScanner with StrictLog
                                replicas.map(_.getSocketAddress).toSet)
     }
   }
+
+  def unwrapTokenRanges(wrappedRanges : Seq[TokenRange] ): HashSet[TokenRange] = {
```

Comment:
How about this instead?

    def unwrapTokenRanges(wrappedRanges: Seq[TokenRange]): Seq[TokenRange] =
      wrappedRanges.flatMap(_.unwrap().asScala.toSeq)